### DISCUSSION TITLE
Mouse theme not working in linux

### DISCUSSION THREAD

**Discussion by YujanSubedi** [ghostty-org/ghostty#3683] [Q&A] [answered]
https://github.com/user-attachments/assets/7ed2b0ba-fcc1-4adc-a4ef-2798b74f34ef

**Comment by 00-kat** [comment, marked as answer]
If you're in the Ghostty discord server, try checking out some of these help threads: https://discord.com/channels/1005603569187160125/1322451477297500210/1322456392258551860 https://discord.com/channels/1005603569187160125/1300810230703194132.

If you're not, the tl;dr was that the cursor they set wasn't properly capitalised; they had to change their cursor from `bibata-modern-ice` to `Bibata-Modern-Ice`.  No one knows why *only* Ghostty is affected.

Edit: also, just to be 100% certain, make sure to try <https://wiki.archlinux.org/title/Cursor_themes#GNOME> and <https://wiki.archlinux.org/title/Cursor_themes#GTK> and possibly even <https://wiki.archlinux.org/title/Cursor_themes#The_default_cursor_theme>.

Edit 2: that person also found that something was resetting the cursor theme name to the lower-case version, so you may need to add setting that to a SystemD service/a cron job/your shell rc.
